### ISSUE TITLE
Links with underscores are not rendered correctly

### ISSUE THREAD

**Issue by ciyer** [SwissDataScienceCenter/renku-ui#2308]
## Description

In certain circumstances, a link that includes a underscores gets treated as markdown to italicize text.  

## Reproduce

This has been seen in the dataset description field, it may appear in other places as well.
Input the following as the dataset description
```
[https://kedro.readthedocs.io/en/stable/tutorial/set_up_data.html](https://kedro.readthedocs.io/en/stable/tutorial/set_up_data.html)

.renku/dataset_images/5b5008c5a6f2415393cec4951f64e226/0.png
```

With this the `up` will be rendered as _up_.

## Task

* Find out if there are places other than dataset description where this is happening
* Fix the problem

The most likely candidate for the source of the problem is the markdown renderer we use. There is a new major version of https://www.npmjs.com/package/showdown, so that would be a good place to start.

**Comment by tomashco** [comment]
Adding literalMidWordUnderscores in showdown config solves the issue.

I've opened the following PR [https://github.com/SwissDataScienceCenter/renku-ui/pull/2371](https://github.com/SwissDataScienceCenter/renku-ui/pull/2371)

**Comment by lorenzo-cavazzi** [comment]
Thank you @tomashco for your contribution to this issue and #1719. We have merged your commit through #2374, and it's included in the latest UI version freshly tagged; that will hit renkulab.io in the next few days.

**Comment by tomashco** [comment]
@lorenzo-cavazzi @ciyer,
thank you for the opportunity to help!
